hugo-modules/core/utils/asset/video.html

```diff
@@ -26,9 +26,10 @@
 {{- $autoplay := $params.autoplay | default true -}}
 {{- $playsinline := $params.playsinline | default true -}}
 {{- $controls := $params.controls | default false -}}
-{{- $poster := $params.poster | default false -}}
+{{/* Use transparent base64, the real poster should be loaded as <picture> element */}}
+{{- $poster := "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" -}}
```

Comment:
why not skip the poster attribute completely?

Reply:
Good question. I asked myself the same question (see TBD). The poster is not a mandatory attribute, but there was some reason... maybe the background is black when the video is not yet loaded? I don't know. I will try to find out and take it out for now.